src/Normalizer/TeamFromUuidDenormalizer.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Normalizer;
+
+use ApiPlatform\Core\Exception\ItemNotFoundException;
+use App\Entity\Team\Team;
+use App\Repository\Team\TeamRepository;
+use Ramsey\Uuid\Uuid;
+use Symfony\Component\Serializer\Normalizer\DenormalizerInterface;
+
+class TeamFromUuidDenormalizer implements DenormalizerInterface
```

Comment:
I think we don't need this denormalizer, ApiPlatform's denormalizer will do the job.